README.adoc

```diff
@@ -99,6 +99,23 @@ To be able to execute JUnit5 Tests with IntellIJ, you need at *least IntellIJ 20
 ==== Use the Selenium Grid
 
 [source,shell]
+
+
+[NOTE]
+--
+To run tests on *PLF 5.2.x* and *PLF 5.1.x* you must ignore *Answer* TC by adding :
+
+ `-Dselenide.test.tags.exclude=answer \`
```

Comment:
Isn't there a way to disable some tests by default ?

Reply:
yes I added this properties to pom.xml 
<selenide.test.tags.exclude>answer</selenide.test.tags.exclude>

Reply:
ok.
Is it necessary to say "*Answer* TC are disabled" ?

Reply:
no, just to inform that answer tests will not be executed.

Reply:
so you prefer to keep it or remove it ?

Reply:
I removed it.